.github/workflows/liam-erd-sample-e2e_tests.yml

```
@@ -0,0 +1,149 @@
+      matrix:
+        browser: [chromium, 'Mobile Safari']
+      fail-fast: false
+    permissions:
+      contents: read
+      statuses: write
+    defaults:
+      run:
+        working-directory: 'frontend/internal-packages/e2e'
+    env:
+      CI: true
+      URL: ${{ steps.snapshot.outputs.target_url }}
```

actionlint (1.7.7)

context "steps" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)